src/coreclr/nativeaot/Runtime/thread.h

```diff
@@ -210,6 +210,9 @@ class Thread : private ThreadBuffer
 #ifndef DACCESS_COMPILE
     void                SetThreadStressLog(void * ptsl);
 #endif // DACCESS_COMPILE
+    void                EnterCantAllocRegion();
+    void                LeaveCanntAllocRegion();
```

Comment:
```suggestion
    void                LeaveCantAllocRegion();
```

Also in the name of the field.

Wondering if we should just name it NoAllocRegion?

Reply:
I forgot to mention that this is copy of CoreCLR logic to deal with the same issue. It is called CantAlloc in CoreCLR. I tried to keep as similar as possible to make eventual unification easier.

Reply:
Thanks for catching the typo!